mlflow/cli/traces.py

```diff
@@ -231,17 +231,17 @@ def commands():
 )
 def search_traces(
     experiment_id: str,
-    filter_string: str | None,
-    max_results: int,
-    order_by: str | None,
-    page_token: str | None,
-    run_id: str | None,
-    include_spans: bool,
-    model_id: str | None,
-    sql_warehouse_id: str | None,
-    output: str,
-    extract_fields: str | None,
-    verbose: bool,
+    filter_string: str | None = None,
+    max_results: int = 100,
+    order_by: str | None = None,
+    page_token: str | None = None,
+    run_id: str | None = None,
+    include_spans: bool = True,
+    model_id: str | None = None,
+    sql_warehouse_id: str | None = None,
+    output: str = "table",
+    extract_fields: str | None = None,
+    verbose: bool = False,
```

Comment:
Can we add descriptions for these args? In particular, I'm concerned that coding agents will try to specify random combinations of args and include things like `sql_warehouse_id` when they aren't needed.